main_exercise10.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+from exercise10 import calculate_subtotal_for_three_items, tax_rate_to_decimal, total_tax_for_three_items, calculate_grand_total
+
+TAX_RATE = 5.5
+
+def main():
```

Comment:
@paulghaddad pylint says I have too many local variables at line 13. What does this mean and refer to??